pages/code-of-conduct.js

```diff
@@ -1,3 +1,118 @@
+import RootLayout from "@/app/layout";
+
+export const metadata = {
```

Comment:
@Amanya-Brian this is going to break the app build

Reply:
![image](https://user-images.githubusercontent.com/65954740/226826071-bcf136a3-3f93-4be8-8cb6-acda4e75ed8c.png)